### PR TITLE
Added new resouces, association and views

### PR DESCRIPTION
**Pull request by TanyaPushchinska**

### CODE REVIEW

File: app/controllers/cars_controller.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  def new
+    @car = Car.new
+  end
+
+  def edit; end
+
+  def create
+    @car = Car.new(car_params)
+
+    respond_to do |format|
```
Comment (Ckazo4nik) [inline app/controllers/cars_controller.rb]: write without respond to

File: app/controllers/cars_controller.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      if @car.save
+        format.html { redirect_to @car, notice: 'Car was successfully created.' }
+      else
+        format.html { render :new }
+      end
+    end
+  end
+
+  def update
+    respond_to do |format|
```
Comment (Ckazo4nik) [inline app/controllers/cars_controller.rb]: write without respond to

File: app/controllers/cars_controller.rb [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        format.html { redirect_to @car, notice: 'Car was successfully updated.' }
+      else
+        format.html { render :edit }
+      end
+    end
+  end
+
+  def destroy
+    @car.destroy
+    respond_to do |format|
```
Comment (Ckazo4nik) [inline app/controllers/cars_controller.rb]: write without respond to

File: app/models/car.rb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+class Car < ApplicationRecord
+  belongs_to :train
+  enum kind: %i[купейний плацкартний]
```
Comment (Ckazo4nik) [inline app/models/car.rb]: change to english

File: app/views/trains/_form.html.erb [hunk 5]
```diff
@@ -16,6 +16,11 @@
     <%= form.collection_select(:railway_station_id, RailwayStation.all, :id, :title) %>
   </div>
 
+  <div class="field">
+    <%= form.label :cars, 'Car' %>
+    <%= form.collection_select(:car_id, Car.all, :id, :number) %>
```
Comment (Ckazo4nik) [inline app/views/trains/_form.html.erb]: I have the opportunity to select three cars?

File: app/views/trains/show.html.erb [hunk 6]
```diff
@@ -1,7 +1,7 @@
 <p id="notice"><%= notice %></p>
 
 <p>
-  <strong>Number:</strong>
+  <strong>Поїзд №</strong>
```
Comment (Ckazo4nik) [inline app/views/trains/show.html.erb]: change to english

File: db/migrate/20190826133124_add_car_id_to_trains.rb [hunk 7]
```diff
@@ -0,0 +1,5 @@
+class AddCarIdToTrains < ActiveRecord::Migration[5.2]
+  def change
+    add_belongs_to :trains, :car
```
Comment (Ckazo4nik) [inline db/migrate/20190826133124_add_car_id_to_trains.rb]: ???

Comment (TanyaPushchinska) [reply db/migrate/20190826133124_add_car_id_to_trains.rb]: so it was in the video

File: app/views/trains/_form.html.erb (hunk 5 above)
Comment (TanyaPushchinska) [reply app/views/trains/_form.html.erb]: no. separately.

File: app/controllers/cars_controller.rb (hunk 3 above)
Comment (TanyaPushchinska) [reply app/controllers/cars_controller.rb]: why?

File: db/migrate/20190826133124_add_car_id_to_trains.rb (hunk 7 above)
Comment (TanyaPushchinska) [reply db/migrate/20190826133124_add_car_id_to_trains.rb]:
```class AddRouteIdToTrains < ActiveRecord::Migration[5.2]
  def change
    add_belongs_to :trains, :route
  end
end```

File: app/models/car.rb [hunk 8]
```diff
@@ -0,0 +1,4 @@
+class Car < ApplicationRecord
```
Comment (Ckazo4nik) [inline app/models/car.rb]: Add some validation

File: db/migrate/20190826133124_add_car_id_to_trains.rb (hunk 7 above)
Comment (Ckazo4nik) [reply db/migrate/20190826133124_add_car_id_to_trains.rb]: Yes route, because train belongs to route!!!

Comment (TanyaPushchinska) [reply db/migrate/20190826133124_add_car_id_to_trains.rb]: Car belongs to train too

File: app/models/car.rb (hunk 8 above)
Comment (TanyaPushchinska) [reply app/models/car.rb]: Done

File: app/views/trains/_form.html.erb (hunk 5 above)
Comment (Ckazo4nik) [reply app/views/trains/_form.html.erb]: remove

File: app/controllers/trains_controller.rb [hunk 9]
```diff
@@ -44,6 +44,6 @@ def set_train
   end
 
   def train_params
-    params.require(:train).permit(:number, :route_id, :railway_station_id, car_ids: [])
+    params.require(:train).permit(:number, :route_id, :railway_station_id, :car_id)
```
Comment (Ckazo4nik) [inline app/controllers/trains_controller.rb]: Why an alone car? Does the train have only one car?

Comment (Ckazo4nik) [reply app/controllers/trains_controller.rb]: remove car_id

File: app/models/ticket.rb [hunk 10]
```diff
@@ -2,6 +2,14 @@
 
 class Ticket < ApplicationRecord
   belongs_to :train
-  belongs_to :user
-  validates :start_station, :end_station, :full_name, presence: true
+  belongs_to :railway_station
+  # validates :start_station, :end_station, :full_name, presence: true
+
+  def start_station
```
Comment (Ckazo4nik) [inline app/models/ticket.rb]: DRY!!!

File: app/views/tickets/index.html.erb [hunk 11]
```diff
@@ -0,0 +1,31 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Продані квитки</h1>
```
Comment (Ckazo4nik) [inline app/views/tickets/index.html.erb]: English!

File: app/models/route.rb [hunk 12]
```diff
@@ -5,12 +5,4 @@ class Route < ApplicationRecord
   has_many :railway_stations, through: :railway_stations_routes
   has_many :trains
   validates :title, presence: true
-
-  def start_station
```
Comment (Ckazo4nik) [inline app/models/route.rb]: How you get stations names?

**Pull request merged**